cmd/serverless/main.go

```diff
@@ -233,6 +236,18 @@ func runAgent(ctx context.Context, stopCh chan struct{}) (err error) {
 		extraTags = append(extraTags, dsdTags...)
 	}
 
+	// adaptive flush configuration
+	if v, exists := os.LookupEnv(flushStrategyEnvVar); exists {
```

Comment:
Why not use the config package ?

Reply:
It is not yet declared in the `config` package (this one and some others from the initial implementation) 🙇 
However that is definitely the goal before GA, I'm waiting for this PR to be merged: https://github.com/DataDog/datadog-agent/pull/7313 which starts using the `config` package to also switch to using config + supporting configuration file.